src/v2/Apps/Settings/Routes/EditSettings/useUpdateSettingsInformation.ts

```diff
@@ -28,6 +28,9 @@ export const useUpdateSettingsInformation = () => {
             $input: UpdateMyProfileInput!
           ) {
             updateMyUserProfile(input: $input) {
+              me {
+                ...SettingsEditSettingsInformation_me
+              }
```

Comment:
This updates the `me` object after mutation and essentially resets the field, re-hiding the password input. Previously this would also break repeated form submissions because if you try to submit the _correct_ password _without_ an email change, Gravity will error 😅